trio/_tests/test_exports.py

```diff
@@ -519,4 +541,4 @@ def test_classes_are_final() -> None:
             if name.endswith("Statistics"):
                 continue
 
-            assert isinstance(class_, _util.Final)
+            assert class_is_final(class_), f"{class_.__module__}.{class_.__qualname__}"
```

Comment:
Hm, see previous comment, would rather this and that be consistent on how much debug info they provide!